### PR TITLE
(AAP-29731) SME review on draft 2

### PR DESCRIPTION
**Pull request by g-murray**
([AAP-29731](https://issues.redhat.com/browse/AAP-29731)) - Review Saas Documentation - PM & Engg

PM and Engg have conducted a review of the full doc (draft 2). This PR is applying the feedback from the review. 

[Draft 2](https://docs.google.com/document/d/1Ythjg5CBAtrV-0BS2-qGuWhIIHXBqnDM3HLh2-ybf54/edit)

### CODE REVIEW

File: stories/topics/con-saas-limited-support-status.adoc [hunk 1]
```diff
@@ -9,7 +9,8 @@ A deployment can return to a fully-supported status if you remediate the violati
...
+If you have not yet configured an execution plane, or have an execution plane in a degraded state, you must correct issues with the execution plane prior to support for automation being provided.
 If you do not maintain the execution plane with supported dependencies:: 
-Both Red{nbsp}Hat Enterprise Linux (RHEL) and OpenShift-based execution planes will require maintenance and upgrades to ensure that those resources maintain the minimum supported versions for {PlatformNameShort} dependencies. Upgrading these resources can be achieved through different models; Ansible may be used for patching, Red Hat Satellite, and DNF automatic are a few examples.
-Ensuring that the OS, cluster, and receptor resources are up-to-date with supported {PlatformNameShort} versions will reduce a support dependency impact.
+Both Red{nbsp}Hat Enterprise Linux (RHEL) and OpenShift-based execution planes require maintenance and upgrades to ensure that those resources maintain the minimum supported versions for {PlatformNameShort} dependencies.
+Upgrading these resources can be achieved through different models; Ansible may be used for patching, Red Hat Satellite, and DNF automatic are a few examples.
```
Comment (ariordan-redhat) [inline stories/topics/con-saas-limited-support-status.adoc:15]: Maybe use active voice here if you can.
And can you rephrase "Ansible may be used for patching, Red Hat Satellite, and DNF automatic are a few examples."

File: stories/topics/con-saas-shape.adoc [hunk 2]
```diff
@@ -10,15 +10,13 @@ Use the following to help guide your automation mesh implementation model:
 Hop nodes typically require minimum resources. 
 The shape of a hop node depends on the number of execution nodes connected to it. 
 Typically a virtual machine (VM) with 2 vCPUs and 2 GB RAM can route traffic for 2-4 execution nodes.   
-** Execution Nodes: {PlatformNameShort} minimum requirements are 4 vCPUs and 8 GB RAM for execution nodes. 
-** {PlatformNameShort} performance and scale testing showed that 2 m4.xlarge execution nodes could effectively automate against 200 managed active nodes constantly with enough remaining CPU and RAM as a buffer to prevent job failures in the event of a spike on those machines. 
-You can use this as a baseline to consider the right machine shapes for your environment when dealing with a CPU-heavy consistent automation load.
-You must monitor and tune the execution plane based on the needs of your organization’s automation.
+** For help with configuring your automation mesh see
+link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_for_operator-based_installations/index[Red Hat Ansible Automation Platform Automation Mesh for operator-based installations].
```
Comment (ariordan-redhat) [inline stories/topics/con-saas-shape.adoc:14]: What version of AAP will SaaS use?
Maybe create a Jira to update the {PlatformVers} when it becomes necessary
I'm concerned that it might clash with Azure moving to 2.5. We can discuss this in the Clouds / docs sync

File: stories/topics/con-saas-shape.adoc [hunk 3]
```diff
@@ -10,15 +10,13 @@ Use the following to help guide your automation mesh implementation model:
...
 Most clouds and hypervisors enable the ability to change shapes with minimal downtime.
-* If you require CPU or RAM intensive automation, you will require larger machine shapes.
+* If you require CPU or RAM intensive automation, you need larger machine shapes.
 * If you have automation that spans multiple locations (such as geographies or other clouds), create a mesh with nodes that enable connectivity to those locations.
 * You can consider the use of different CPU architectures, like ARM, and reserved instances to reduce the cost of the execution plane.
-* If you are interested in configuring redundancy in the automation mesh, you  should configure at least two mesh nodes of the same shape in the same region, but in different availability zones, with each machine connected to both mesh ingress hop nodes.
+* If you are interested in configuring redundancy in the automation mesh, you can configure at least two mesh nodes of the same shape in the same region, but in different availability zones, with each machine connected to both mesh ingress hop nodes.
```
Comment (ariordan-redhat) [inline stories/topics/con-saas-shape.adoc:20]:
```suggestion
* If you want to configure redundancy in the automation mesh, you can configure at least two mesh nodes of the same shape in the same region but in different availability zones, with each machine connected to both mesh ingress hop nodes.
```

File: stories/topics/proc-saas-set-up-private.adoc [hunk 4]
```diff
@@ -1,29 +1,52 @@
...
+If you have already accepted your offer you can find it in the *Manage subscriptions* page.
+====
++
 . CLick the URL under the *Offer ID* column. This redirects you to the *Offer Selection* page. 
 . Click btn:[Create contract].
 . Click btn:[Click Set up your account]. This redirects you to {RHSSO} where you need to:
 .. Create or sign into your Red{nbsp}Hat account.
 .. Connect your AWS account to the Red{nbsp}Hat account.
-... If this is your first time connecting your accounts you will need to accept the terms and conditions and click btn:[Connect accounts].
+... If this is your first time connecting your accounts you need to accept the terms and conditions and click btn:[Connect accounts].
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-private.adoc:28]:
```suggestion
... If this is your first time connecting your accounts, you must accept the terms and conditions and click btn:[Connect accounts].
```

File: stories/topics/proc-saas-set-up-private.adoc [hunk 5]
```diff
@@ -1,29 +1,52 @@
...
-This account ID must be the account ID that purchased the offer from the AWS marketplace. The system will not recognize associated or nested accounts.
-. Once your AWS ID is validated click btn:[Next].
+This account ID must be the account ID that purchased the offer from the AWS marketplace. The system does not recognize associated or nested accounts.
+. After your AWS ID is validated click btn:[Next].
 . Select your desired region and click btn:[Next].
 . Click btn:[Provision].
 
-This redirects you to the *Instances* page which displays all the details of the instance you created. From here you can confirm if your instance is *Assigned* or still in a *Pending* state.
+This redirects you to the *Environment details* page of the *Instances* tab which displays all the details of the instance you created.
+From here you can view if your instance is *Ready* or still in a *Provision in progress* state.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-private.adoc:38]:
```suggestion
From here you can view whether your instance is *Ready* or still in a *Provision in progress* state.
```

File: stories/topics/proc-saas-set-up-private.adoc [hunk 6]
```diff
@@ -1,29 +1,52 @@
...
+
+Provisioning your environment can take up to two hours.
+When your environment is ready a confirmation email is sent to the email address linked to your account. 
+
+[IMPORTANT]
+====
+You need to update your password as soon as possible.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-private.adoc:45]:
```suggestion
Update your password as soon as possible.
```

File: stories/topics/proc-saas-set-up-private.adoc [hunk 7]
```diff
@@ -1,29 +1,52 @@
...
+Your confirmation email includes a link that shows your admin password.
+Use this password to log-in and change your password.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-private.adoc:47]:
```suggestion
Use this password to log in and change your password.
```
From the IBM style guide

File: stories/topics/proc-saas-set-up-private.adoc [hunk 8]
```diff
@@ -1,29 +1,52 @@
...
+This link can only be accessed once, so ensure you are ready before clicking it.  
+
+In the event the email is not received or initial admin password expires, you need to submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-private.adoc:50]:
```suggestion
If you did not receive the email or the initial admin password has expired, you must submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.
```

File: stories/topics/proc-saas-set-up-public.adoc [hunk 9]
```diff
@@ -22,12 +32,26 @@ As an AWS customer you can search for and subscribe to {SaaSonAWS} from the AWS
...
-. Once your AWS ID is validated click btn:[Next].
+The system does not recognize associated or nested accounts.
+. After your AWS ID is validated click btn:[Next].
 . Select your desired region.
 . Click btn:[Provision].
 
-This redirects you to the *Instances* page which displays all the details of the instance you created. From here you can confirm if your instance is *Assigned* or still in a *Pending* state.
\ No newline at end of file
+This redirects you to the *Environment details* page of the *Instances* tab which displays all the details of the instance you created.
+From here you can view if your instance is *Ready* or still in a *Provision in progress* state.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-public.adoc:44]:
```suggestion
From here you can view whether your instance is *Ready* or still in a *Provision in progress* state.
```

File: stories/topics/proc-saas-set-up-public.adoc [hunk 10]
```diff
@@ -22,12 +32,26 @@ As an AWS customer you can search for and subscribe to {SaaSonAWS} from the AWS
...
+
+Provisioning your environment can take up to two hours.
+When your environment is ready a confirmation email is sent to the email address linked to your account. 
+
+[IMPORTANT]
+====
+You need to update your password as soon as possible.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-public.adoc:51]:
```suggestion
Update your password as soon as possible.
```

File: stories/topics/proc-saas-set-up-public.adoc [hunk 11]
```diff
@@ -22,12 +32,26 @@ As an AWS customer you can search for and subscribe to {SaaSonAWS} from the AWS
...
+Your confirmation email includes a link that shows your admin password.
+Use this password to log-in and change your password.
+This link can only be accessed once, so ensure you are ready before clicking it. 
+
+In the event the email is not received or initial admin password expires, you need to submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.
```
Comment (ariordan-redhat) [inline stories/topics/proc-saas-set-up-public.adoc:56]:
```suggestion
In the event the email is not received or initial admin password expires, you need to submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.
```
If you did not receive the email or the initial admin password has expired, you must submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.

File: titles/saas-aws/master.adoc [hunk 12]
```diff
@@ -10,5 +10,7 @@ include::title-attributes.adoc[]
 [id="{ProjectNameID}-{context}"]
 = Red Hat Ansible Automation Platform Service on AWS
 
+The document outlines the service definition and set up  for {SaaSonAWS}.
```
Comment (ariordan-redhat) [inline titles/saas-aws/master.adoc:13]:
```suggestion
This document outlines the service definition and set up  for {SaaSonAWS}.
```

File: stories/topics/con-saas-shape.adoc (hunk 2 above)
Comment (g-murray) [reply stories/topics/con-saas-shape.adoc:14]: Good question, I assumed 2.5 but I shouldn't be assuming anything.

**Review by ariordan-redhat — APPROVED** [review]
LGTM
